src/Spatial/Euclidean/PolyLine2D.cs

```diff
@@ -15,7 +16,7 @@ public class PolyLine2D : IEquatable<PolyLine2D>
         /// <summary>
         /// Internal storage for the points
         /// </summary>
-        private readonly List<Point2D> points;
+        private readonly ReadOnlyCollection<Point2D> points;
```

Comment:
I am in a trajectory where I try to remove all redundant ```this.``` qualifiers from all classes. So please rename this private field to ```_points``` and remove all ```this.``` from these files.